installer/installer.py

```diff
@@ -522,6 +524,98 @@ def cmd_install_netmodules(public_slave=False):
     return
 
 
+def cmd_install_cni(public_slave=False):
+    """
+    Install Calico's CNI plugin.  A successful completion of the task
+    indicates successful installation
+
+    :param public_slave: Flag indicating if this is a public slave.
+    """
+    # Load the current Calico install info for Docker, and the current Docker
+    # daemon configuration.
+    install_config = load_config(CNI_INSTALL_CONFIG)
+
+    # Before starting the install, check that we are able to locate the agent
+    # process - if not there is not much we can do here.
+    if not install_config:
+        _log.debug("Have not started installation yet")
```

Comment:
Dan - as discussed yesterday - I think we need to make sure we can get sufficient logs from a customer if something goes wrong in the field.  If we can't turn on logging, or adjust the level in the field without a code change then we need to make sure we write enough logs at a level that is logged.

These scripts aren't run very often and only during install time - so we probably can get away with logging most control flow.   WDYT?